cmd/provisioner-localpv/app/quota_hostpath.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2019 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+This code was taken from https://github.com/rancher/local-path-provisioner
+and modified to work with the configuration options used by OpenEBS
+*/
+
+package app
+
+import (
```

Comment:
Can you confirm this we can use that commands in helper pod?
I also think it is unnecessary to launch a pod.

Reply:
No, I meant creating a new pod is necessary in cases where the user's want quota. If the user doesn't want to enforce quota then we will not launch the quota pod. What I meant is shift the code of this file in the helper_hostpath.go file itself and instead of creating a new function for deleting the pod you can call the same function and for creating also same function can be called with some additional parameters added if needed.